plugdj/tests/__init__.py

```diff
@@ -18,7 +19,7 @@ def test_chat():
 
 def test_send_nonstring_chat():
     assert is_ok(p.join_room(creds.room))
-    p.send_chat(25)
+    tools.assert_raises(TypeError, p.send_chat, 25)
```

Comment:
I don't really see the need for this to work, but it could be made to do so.

Reply:
@bryant, is there a use-case for sending integers that I'm missing?

Reply:
My memory is a little fuzzy here. I think that at the time, I simply wanted to see how the plug server would react to non-string messages. We could XFAIL this for now, if it no longer works.

Reply:
My memory is a little fuzzy here. I think that at the time, I simply wanted to see how the plug server would react to non-string messages. We could XFAIL this for now, if it no longer works.

Reply:
Sounds good, leaving it as is seems like the way to go. Anything else you'd like to see frm the PR to get it merged?

Reply:
Optimally, we would chunk this into tightly contained patches, or separate PRs altogether. If that seems like a hassle, I could do it (with full credit attributed to you). However, I am a little bit busy these few days. So I will get to this in a couple of days.

Reply:
I'll split things up into sep. PRs